tags/tag-workloads-foundation/charter.md

```diff
@@ -1 +1,57 @@
-Charter content here
\ No newline at end of file
+# CNCF TAG Workloads Foundation
+
+## Mission
+
+To define and advance practices and standards for fundamental cloud native workload execution environments and their related lifecycle management within cloud native systems, applications, and architectures. This supports the CNCF's technical vision by addressing critical problems faced by adopters and contributing to a robust cloud native ecosystem.
+
+## Scope
+
+This TAG covers the following sub-domains and topics within its scope:
+
+- Containers
+- OS
+- Runtime
+- Virtual Machines
+- Serverless
+- Web Assembly
+- Batch
```

Comment:
Have we made an explicit decision to exclude databases or has it been omitted in error?

Reply:
Nice Catch.